client/landing/stepper/declarative-flow/link-in-bio.ts

```diff
@@ -15,12 +22,92 @@ export const linkInBio: Flow = {
 		return [
 			'intro',
 			'linkInBioSetup',
+			'linkInBioProcessingStep',
 			...( isEnabled( 'signup/launchpad' ) ? [ 'launchpad' ] : [] ),
 		] as StepPath[];
 	},
 
 	useStepNavigation( _currentStep, navigate ) {
+		const { getNewSite } = useSelect( ( select ) => select( SITE_STORE ) );
+		const siteSlugParam = useSiteSlugParam();
+		const { setPendingAction, setProgress, setProgressTitle } = useDispatch( ONBOARD_STORE );
+		const { saveSiteSettings } = useDispatch( SITE_STORE );
+		const site = useSite();
+		const { mutateAsync: setSiteLogo } = useSiteLogoMutation( site?.ID );
+		const { __ } = useI18n();
+
+		const { getState } = useSelect( ( select ) => select( ONBOARD_STORE ) );
+
+		const base64ImageToBlob = ( base64String: string ) => {
+			// extract content type and base64 payload from original string
+			const pos = base64String.indexOf( ';base64,' );
+			const type = base64String.substring( 5, pos );
+			const b64 = base64String.substr( pos + 8 );
+
+			// decode base64
+			const imageContent = atob( b64 );
+
+			// create an ArrayBuffer and a view (as unsigned 8-bit)
+			const buffer = new ArrayBuffer( imageContent.length );
+			const view = new Uint8Array( buffer );
+
+			// fill the view, using the decoded base64
+			for ( let n = 0; n < imageContent.length; n++ ) {
+				view[ n ] = imageContent.charCodeAt( n );
+			}
+
+			// convert ArrayBuffer to Blob
+			const blob = new Blob( [ buffer ], { type: type } );
+
+			return blob;
+		};
+
+		const flowEnd = () => {
+			if ( site ) {
```

Comment:
We don't have site, when we submit from `linkInBioSetup`. So we should remove it from here, as well as inside the setup step :)